### ISSUE TITLE
Magento 2.1.7 not getting correct urls

### ISSUE THREAD

**Issue by hellekim** [Iazel/magento2-regenurl#17]
I used this succesfully with 2.1.5, but with 2.1.7 nothing is changing when I run this code.

Any thoughts?

**Comment by Culth** [comment]
I think you have to set the product store id in all cases. At least this fixed the issue for me.

**Comment by hellekim** [comment]
I received this reply to my email, but can't find from here, so it was:

> I think you have to set the product store id in all cases. At least this fixed the issue for me.

I used this command: php bin/magento iazel:regenurl -s1 and changed the end to -s2 etc. for different shops. Still one shop only has broken urls and it's super frustrating (althought this should be fixed by Magento, not @Iazel of course.

**Comment by ghost** [comment]
Yep i deleted the comment since i noticed this was probably a different issue. Since i upgraded to 2.1.7 i had the issue that the plugin sets the URLs for the wrong scopes. I also use a modified Version of lazels module. What Kind of broken URL´s are you talking about by the way? I wouldnt call myself a magento specialist but maybe i can help.

Regards

Gesendet von Mail<https://go.microsoft.com/fwlink/?LinkId=550986> für Windows 10

Von: hellekim<mailto:notifications@github.com>
Gesendet: Donnerstag, 29. Juni 2017 18:27
An: Iazel/magento2-regenurl<mailto:magento2-regenurl@noreply.github.com>
Cc: Culth<mailto:schienkey@hotmail.de>; Comment<mailto:comment@noreply.github.com>
Betreff: Re: [Iazel/magento2-regenurl] Magento 2.1.7 not getting correct urls (#17)


I received this reply to my email, but can't find from here, so it was:

I think you have to set the product store id in all cases. At least this fixed the issue for me.

I used this command: php bin/magento iazel:regenurl -s1 and changed the end to -s2 etc. for different shops. Still one shop only has broken urls and it's super frustrating (althought this should be fixed by Magento, not @Iazel<https://github.com/iazel> of course.

**Comment by hellekim** [comment]
OK - Thanks! My issue was down to Mageworx SEO extension which was messing with the URLs. Disabled that, run this and all was ok again.
